ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-555] Add which other savings page

## Description of change
Adds ‘Which other savings does your client have inside or outside the UK?’ screen, adding validation and linking to current savings flow

## Link to relevant ticket
[CRIMAPP-555](https://dsdmoj.atlassian.net/browse/CRIMAPP-555)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:
<img width="992" alt="Screenshot 2024-02-29 at 09 03 19" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/1e2d4e59-5673-412f-9979-3842fd6ea29d">

<img width="992" alt="Screenshot 2024-02-29 at 09 29 23" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/51488cdf-3fc9-4f84-9ba2-fd16d754e0e1">

## How to manually test the feature
Pull down this branch
Start an application 
Go to `/steps/capital/which_savings_does_client_have`
Follow the flow to add a savings account
When on the savings summary page, select yes to add another saving
This should route you to the ‘Which other savings does your client have inside or outside the UK?’ screen
Click save and continue and check the validation error is in place 
Select a saving account option and select save and continue and confirm the validation error is not shown and that you are taken to the page to fill in the saving account details

[CRIMAPP-555]: https://dsdmoj.atlassian.net/browse/CRIMAPP-555?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
